<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.17">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Baxter Worthing">

<title>Adaptive Introgression Shapes the Pan-genome of Populus Hybrid Zones</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for citations */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
  margin-bottom: 0em;
}
.hanging-indent div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}</style>


<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-html/quarto.js" type="module"></script>
<script src="site_libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting-da03d714c310c60cec1d83284c92da2c.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap-3633567391c49fee829cf872490df673.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<script src="site_libs/quarto-contrib/glightbox/glightbox.min.js"></script>
<link href="site_libs/quarto-contrib/glightbox/glightbox.min.css" rel="stylesheet">
<link href="site_libs/quarto-contrib/glightbox/lightbox.css" rel="stylesheet">
<script async="" src="https://hypothes.is/embed.js"></script>
<script>
  window.document.addEventListener("DOMContentLoaded", function (_event) {
    document.body.classList.add('hypothesis-enabled');
  });
</script>


<meta name="citation_title" content="Adaptive Introgression Shapes the Pan-genome of Populus Hybrid Zones">
<meta name="citation_abstract" content="Poplar trees...
">
<meta name="citation_author" content="Baxter Worthing">
<meta name="citation_language" content="en">
<meta name="citation_reference" content="citation_title=The plant genome integrative explorer resource: PlantGenIE. org;,citation_author=David Sundell;,citation_author=Chanaka Mannapperuma;,citation_author=Sergiu Netotea;,citation_author=Nicolas Delhomme;,citation_author=Yao-Cheng Lin;,citation_author=Andreas Sjodin;,citation_author=Yves Van de Peer;,citation_author=Stefan Jansson;,citation_author=Torgeir R Hvidsten;,citation_author=Nathaniel R Street;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_publisher=JSTOR;">
<meta name="citation_reference" content="citation_title=BEDTools: A flexible suite of utilities for comparing genomic features;,citation_author=Aaron R Quinlan;,citation_author=Ira M Hall;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_issue=6;,citation_volume=26;,citation_journal_title=Bioinformatics;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Second-generation PLINK: Rising to the challenge of larger and richer datasets;,citation_author=Christopher C Chang;,citation_author=Carson C Chow;,citation_author=Laurent CAM Tellier;,citation_author=Shashaank Vattikuti;,citation_author=Shaun M Purcell;,citation_author=James J Lee;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=1;,citation_volume=4;,citation_journal_title=Gigascience;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Loter: A software package to infer local ancestry for a wide range of species;,citation_author=Thomas Dias-Alves;,citation_author=Julien Mairal;,citation_author=Michael GB Blum;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_issue=9;,citation_volume=35;,citation_journal_title=Molecular biology and evolution;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Genomic insights into hybrid zone formation: The role of climate, landscape, and demography in the emergence of a novel hybrid lineage;,citation_author=Constance E Bolte;,citation_author=Tommy Phannareth;,citation_author=Michelle Zavala-Paez;,citation_author=Brianna N Sutara;,citation_author=Muhammed F Can;,citation_author=Matthew C Fitzpatrick;,citation_author=Jason A Holliday;,citation_author=Stephen R Keller;,citation_author=Jill A Hamilton;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_issue=14;,citation_volume=33;,citation_journal_title=Molecular ecology;,citation_publisher=Wiley Online Library;">
<meta name="citation_reference" content="citation_title=R: A language and environment for statistical computing;,citation_author=R Core Team;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_fulltext_html_url=https://www.R-project.org/;">
<meta name="citation_reference" content="citation_title=Identification of errors in draft genome assemblies at single-nucleotide resolution for quality assessment and improvement;,citation_author=Kunpeng Li;,citation_author=Peng Xu;,citation_author=Jinpeng Wang;,citation_author=Xin Yi;,citation_author=Yuannian Jiao;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=1;,citation_volume=14;,citation_journal_title=Nature Communications;,citation_publisher=Nature Publishing Group UK London;">
<meta name="citation_reference" content="citation_title=Pangenome graphs and their applications in biodiversity genomics;,citation_author=Simona Secomandi;,citation_author=Guido Roberto Gallo;,citation_author=Riccardo Rossi;,citation_author=Carlos Rodrı́guez Fernandes;,citation_author=Erich D Jarvis;,citation_author=Andrea Bonisoli-Alquati;,citation_author=Luca Gianfranceschi;,citation_author=Giulio Formenti;,citation_publication_date=2025;,citation_cover_date=2025;,citation_year=2025;,citation_journal_title=Nature Genetics;,citation_publisher=Nature Publishing Group US New York;">
<meta name="citation_reference" content="citation_title=The tomato pan-genome uncovers new genes and a rare allele regulating fruit flavor;,citation_author=Lei Gao;,citation_author=Itay Gonda;,citation_author=Honghe Sun;,citation_author=Qiyue Ma;,citation_author=Kan Bao;,citation_author=Denise M Tieman;,citation_author=Elizabeth A Burzynski-Chang;,citation_author=Tara L Fish;,citation_author=Kaitlin A Stromberg;,citation_author=Gavin L Sacks;,citation_author=others;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_issue=6;,citation_volume=51;,citation_journal_title=Nature genetics;,citation_publisher=Nature Publishing Group US New York;">
<meta name="citation_reference" content="citation_title=The pig pangenome provides insights into the roles of coding structural variations in genetic diversity and adaptation;,citation_author=Zhengcao Li;,citation_author=Xiaohong Liu;,citation_author=Chen Wang;,citation_author=Zhenyang Li;,citation_author=Bo Jiang;,citation_author=Ruifeng Zhang;,citation_author=Lu Tong;,citation_author=Youping Qu;,citation_author=Sheng He;,citation_author=Haifan Chen;,citation_author=others;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=10;,citation_volume=33;,citation_journal_title=Genome Research;,citation_publisher=Cold Spring Harbor Lab;">
<meta name="citation_reference" content="citation_title=Pan-genome analysis highlights the role of structural variation in the evolution and environmental adaptation of asian honeybees;,citation_author=Yancan Li;,citation_author=Jun Yao;,citation_author=Huiling Sang;,citation_author=Quangui Wang;,citation_author=Long Su;,citation_author=Xiaomeng Zhao;,citation_author=Zhenyu Xia;,citation_author=Feiran Wang;,citation_author=Kai Wang;,citation_author=Delong Lou;,citation_author=others;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_issue=2;,citation_volume=24;,citation_journal_title=Molecular ecology resources;,citation_publisher=Wiley Online Library;">
<meta name="citation_reference" content="citation_title=Genomic patterns of structural variation among diverse genotypes of sorghum bicolor and a potential role for deletions in local adaptation;,citation_author=Kittikun Songsomboon;,citation_author=Zachary Brenton;,citation_author=James Heuser;,citation_author=Stephen Kresovich;,citation_author=Nadia Shakoor;,citation_author=Todd Mockler;,citation_author=Elizabeth A Cooper;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=7;,citation_volume=11;,citation_journal_title=G3;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Genomic structural variants constrain and facilitate adaptation in natural populations of theobroma cacao, the chocolate tree;,citation_author=Tuomas Hämälä;,citation_author=Eric K Wafula;,citation_author=Mark J Guiltinan;,citation_author=Paula E Ralph;,citation_author=Claude W Depamphilis;,citation_author=Peter Tiffin;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=35;,citation_volume=118;,citation_journal_title=Proceedings of the National Academy of Sciences;,citation_publisher=National Academy of Sciences;">
<meta name="citation_reference" content="citation_title=Adaptive introgression as a resource for management and genetic conservation in a changing climate;,citation_author=Jill A Hamilton;,citation_author=Joshua M Miller;,citation_publication_date=2016;,citation_cover_date=2016;,citation_year=2016;,citation_issue=1;,citation_volume=30;,citation_journal_title=Conservation Biology;,citation_publisher=Wiley Online Library;">
<meta name="citation_reference" content="citation_title=Adaptive introgression facilitates adaptation to high latitudes in european aspen (populus tremula l.);,citation_author=Martha Rendón-Anaya;,citation_author=Jonathan Wilson;,citation_author=Sæmundur Sveinsson;,citation_author=Aleksey Fedorkov;,citation_author=Joan Cottrell;,citation_author=Mark ES Bailey;,citation_author=Dainis Ruņǵis;,citation_author=Christian Lexer;,citation_author=Stefan Jansson;,citation_author=Kathryn M Robinson;,citation_author=others;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=11;,citation_volume=38;,citation_journal_title=Molecular Biology and Evolution;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Adaptive introgression as a driver of local adaptation to climate in european white oaks;,citation_author=Thibault Leroy;,citation_author=Jean-Marc Louvet;,citation_author=Céline Lalanne;,citation_author=Grégoire Le Provost;,citation_author=Karine Labadie;,citation_author=Jean-Marc Aury;,citation_author=Sylvain Delzon;,citation_author=Christophe Plomion;,citation_author=Antoine Kremer;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=4;,citation_volume=226;,citation_journal_title=New Phytologist;,citation_publisher=Wiley Online Library;">
<meta name="citation_reference" content="citation_title=Adaptive introgression: A plant perspective;,citation_author=Adriana Suarez-Gonzalez;,citation_author=Christian Lexer;,citation_author=Quentin CB Cronk;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_issue=3;,citation_volume=14;,citation_journal_title=Biology letters;,citation_publisher=The Royal Society;">
<meta name="citation_reference" content="citation_title=Fitness consequences of structural variation inferred from a house finch pangenome;,citation_author=Bohao Fang;,citation_author=Scott V Edwards;,citation_publication_date=2024;,citation_cover_date=2024;,citation_year=2024;,citation_issue=47;,citation_volume=121;,citation_journal_title=Proceedings of the National Academy of Sciences;,citation_publisher=National Academy of Sciences;">
<meta name="citation_reference" content="citation_title=Pan-genome of raphanus highlights genetic variation and introgression among domesticated, wild, and weedy radishes;,citation_author=Xiaohui Zhang;,citation_author=Tongjin Liu;,citation_author=Jinglei Wang;,citation_author=Peng Wang;,citation_author=Yang Qiu;,citation_author=Wei Zhao;,citation_author=Shuai Pang;,citation_author=Xiaoman Li;,citation_author=Haiping Wang;,citation_author=Jiangping Song;,citation_author=others;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=12;,citation_volume=14;,citation_journal_title=Molecular Plant;,citation_publisher=Elsevier;">
<meta name="citation_reference" content="citation_title=Structural variation and introgression from wild populations in east asian cattle genomes confer adaptation to local environment;,citation_author=Xiaoting Xia;,citation_author=Fengwei Zhang;,citation_author=Shuang Li;,citation_author=Xiaoyu Luo;,citation_author=Lixin Peng;,citation_author=Zheng Dong;,citation_author=Hubert Pausch;,citation_author=Alexander S Leonard;,citation_author=Danang Crysnanto;,citation_author=Shikang Wang;,citation_author=others;,citation_publication_date=2023;,citation_cover_date=2023;,citation_year=2023;,citation_issue=1;,citation_volume=24;,citation_journal_title=Genome Biology;,citation_publisher=Springer;">
<meta name="citation_reference" content="citation_title=Population structure, stratification, and introgression of human structural variation;,citation_author=Mohamed A Almarri;,citation_author=Anders Bergström;,citation_author=Javier Prado-Martinez;,citation_author=Fengtang Yang;,citation_author=Beiyuan Fu;,citation_author=Alistair S Dunham;,citation_author=Yuan Chen;,citation_author=Matthew E Hurles;,citation_author=Chris Tyler-Smith;,citation_author=Yali Xue;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=1;,citation_volume=182;,citation_journal_title=Cell;,citation_publisher=Elsevier;">
<meta name="citation_reference" content="citation_title=Frequent intra-and inter-species introgression shapes the landscape of genetic variation in bread wheat;,citation_author=Hong Cheng;,citation_author=Jing Liu;,citation_author=Jia Wen;,citation_author=Xiaojun Nie;,citation_author=Luohao Xu;,citation_author=Ningbo Chen;,citation_author=Zhongxing Li;,citation_author=Qilin Wang;,citation_author=Zhuqing Zheng;,citation_author=Ming Li;,citation_author=others;,citation_publication_date=2019;,citation_cover_date=2019;,citation_year=2019;,citation_volume=20;,citation_journal_title=Genome biology;,citation_publisher=Springer;">
<meta name="citation_reference" content="citation_title=Evolutionary genomics of structural variation in asian rice (oryza sativa) domestication;,citation_author=Yixuan Kou;,citation_author=Yi Liao;,citation_author=Tuomas Toivainen;,citation_author=Yuanda Lv;,citation_author=Xinmin Tian;,citation_author=JJ Emerson;,citation_author=Brandon S Gaut;,citation_author=Yongfeng Zhou;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=12;,citation_volume=37;,citation_journal_title=Molecular biology and evolution;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=How important are structural variants for speciation?;,citation_author=Linyi Zhang;,citation_author=Radka Reifová;,citation_author=Zuzana Halenková;,citation_author=Zachariah Gompert;,citation_publication_date=2021;,citation_cover_date=2021;,citation_year=2021;,citation_issue=7;,citation_volume=12;,citation_journal_title=Genes;,citation_publisher=MDPI;">
<meta name="citation_reference" content="citation_title=HiCanu: Accurate assembly of segmental duplications, satellites, and allelic variants from high-fidelity long reads;,citation_author=Sergey Nurk;,citation_author=Brian P Walenz;,citation_author=Arang Rhie;,citation_author=Mitchell R Vollger;,citation_author=Glennis A Logsdon;,citation_author=Robert Grothe;,citation_author=Karen H Miga;,citation_author=Evan E Eichler;,citation_author=Adam M Phillippy;,citation_author=Sergey Koren;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=9;,citation_volume=30;,citation_journal_title=Genome research;,citation_publisher=Cold Spring Harbor Lab;">
<meta name="citation_reference" content="citation_title=Metagenome analysis using the Kraken software suite;,citation_author=Jennifer Lu;,citation_author=Natalia Rincon;,citation_author=Derrick E. Wood;,citation_author=Florian P. Breitwieser;,citation_author=Christopher Pockrandt;,citation_author=Ben Langmead;,citation_author=Steven L. Salzberg;,citation_author=Martin Steinegger;,citation_publication_date=2022-09-28;,citation_cover_date=2022-09-28;,citation_year=2022;,citation_fulltext_html_url=http://dx.doi.org/10.1038/s41596-022-00738-y;,citation_issue=12;,citation_doi=10.1038/s41596-022-00738-y;,citation_volume=17;,citation_language=en;,citation_journal_title=Nature Protocols;">
<meta name="citation_reference" content="citation_title=Minimap2: Pairwise alignment for nucleotide sequences;,citation_author=Heng Li;,citation_publication_date=2018;,citation_cover_date=2018;,citation_year=2018;,citation_issue=18;,citation_volume=34;,citation_journal_title=Bioinformatics;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Weighted minimizer sampling improves long read mapping;,citation_author=Chirag Jain;,citation_author=Arang Rhie;,citation_author=Haowen Zhang;,citation_author=Claudia Chu;,citation_author=Brian P Walenz;,citation_author=Sergey Koren;,citation_author=Adam M Phillippy;,citation_publication_date=2020;,citation_cover_date=2020;,citation_year=2020;,citation_issue=Supplement_1;,citation_volume=36;,citation_journal_title=Bioinformatics;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Fast and accurate long-read alignment with burrows–wheeler transform;,citation_author=Heng Li;,citation_author=Richard Durbin;,citation_publication_date=2010;,citation_cover_date=2010;,citation_year=2010;,citation_issue=5;,citation_volume=26;,citation_journal_title=Bioinformatics;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Pangenome graph construction from genome alignments with Minigraph-Cactus;,citation_author=Glenn Hickey;,citation_author=Jean Monlong;,citation_author=Jana Ebler;,citation_author=Adam M. Novak;,citation_author=Jordan M. Eizenga;,citation_author=Yan Gao;,citation_author=Haley J. Abel;,citation_author=Lucinda L. Antonacci-Fulton;,citation_author=Mobin Asri;,citation_author=Gunjan Baid;,citation_author=Carl A. Baker;,citation_author=Anastasiya Belyaeva;,citation_author=Konstantinos Billis;,citation_author=Guillaume Bourque;,citation_author=Silvia Buonaiuto;,citation_author=Andrew Carroll;,citation_author=Mark J. P. Chaisson;,citation_author=Pi-Chuan Chang;,citation_author=Xian H. Chang;,citation_author=Haoyu Cheng;,citation_author=Justin Chu;,citation_author=Sarah Cody;,citation_author=Vincenza Colonna;,citation_author=Daniel E. Cook;,citation_author=Robert M. Cook-Deegan;,citation_author=Omar E. Cornejo;,citation_author=Mark Diekhans;,citation_author=Daniel Doerr;,citation_author=Peter Ebert;,citation_author=Jana Ebler;,citation_author=Evan E. Eichler;,citation_author=Susan Fairley;,citation_author=Olivier Fedrigo;,citation_author=Adam L. Felsenfeld;,citation_author=Xiaowen Feng;,citation_author=Christian Fischer;,citation_author=Paul Flicek;,citation_author=Giulio Formenti;,citation_author=Adam Frankish;,citation_author=Robert S. Fulton;,citation_author=Shilpa Garg;,citation_author=Erik Garrison;,citation_author=Nanibaa’ A. Garrison;,citation_author=Carlos Garcia Giron;,citation_author=Richard E. Green;,citation_author=Cristian Groza;,citation_author=Andrea Guarracino;,citation_author=Leanne Haggerty;,citation_author=Ira M. Hall;,citation_author=William T. Harvey;,citation_author=Marina Haukness;,citation_author=David Haussler;,citation_author=Simon Heumos;,citation_author=Kendra Hoekzema;,citation_author=Thibaut Hourlier;,citation_author=Kerstin Howe;,citation_author=Miten Jain;,citation_author=Erich D. Jarvis;,citation_author=Hanlee P. Ji;,citation_author=Eimear E. Kenny;,citation_author=Barbara A. Koenig;,citation_author=Alexey Kolesnikov;,citation_author=Jan O. Korbel;,citation_author=Jennifer Kordosky;,citation_author=Sergey Koren;,citation_author=HoJoon Lee;,citation_author=Alexandra P. Lewis;,citation_author=Wen-Wei Liao;,citation_author=Shuangjia Lu;,citation_author=Tsung-Yu Lu;,citation_author=Julian K. Lucas;,citation_author=Hugo Magalhães;,citation_author=Santiago Marco-Sola;,citation_author=Pierre Marijon;,citation_author=Charles Markello;,citation_author=Tobias Marschall;,citation_author=Fergal J. Martin;,citation_author=Ann McCartney;,citation_author=Jennifer McDaniel;,citation_author=Karen H. Miga;,citation_author=Matthew W. Mitchell;,citation_author=Jacquelyn Mountcastle;,citation_author=Katherine M. Munson;,citation_author=Moses Njagi Mwaniki;,citation_author=Maria Nattestad;,citation_author=Sergey Nurk;,citation_author=Hugh E. Olsen;,citation_author=Nathan D. Olson;,citation_author=Trevor Pesout;,citation_author=Adam M. Phillippy;,citation_author=Alice B. Popejoy;,citation_author=David Porubsky;,citation_author=Pjotr Prins;,citation_author=Daniela Puiu;,citation_author=Mikko Rautiainen;,citation_author=Allison A. Regier;,citation_author=Arang Rhie;,citation_author=Samuel Sacco;,citation_author=Ashley D. Sanders;,citation_author=Valerie A. Schneider;,citation_author=Baergen I. Schultz;,citation_author=Kishwar Shafin;,citation_author=Jonas A. Sibbesen;,citation_author=Jouni Sirén;,citation_author=Michael W. Smith;,citation_author=Heidi J. Sofia;,citation_author=Ahmad N. Abou Tayoun;,citation_author=Françoise Thibaud-Nissen;,citation_author=Chad Tomlinson;,citation_author=Francesca Floriana Tricomi;,citation_author=Flavia Villani;,citation_author=Mitchell R. Vollger;,citation_author=Justin Wagner;,citation_author=Brian Walenz;,citation_author=Ting Wang;,citation_author=Jonathan M. D. Wood;,citation_author=Aleksey V. Zimin;,citation_author=Justin M. Zook;,citation_author=Tobias Marschall;,citation_author=Heng Li;,citation_author=Benedict Paten;,citation_publication_date=2023-05-10;,citation_cover_date=2023-05-10;,citation_year=2023;,citation_fulltext_html_url=http://dx.doi.org/10.1038/s41587-023-01793-w;,citation_issue=4;,citation_doi=10.1038/s41587-023-01793-w;,citation_volume=42;,citation_language=en;,citation_journal_title=Nature Biotechnology;">
<meta name="citation_reference" content="citation_title=Variation graph toolkit improves read mapping by representing genetic variation in the reference;,citation_author=Erik Garrison;,citation_author=Jouni Sirén;,citation_author=Adam M Novak;,citation_author=Glenn Hickey;,citation_author=Jordan M Eizenga;,citation_author=Eric T Dawson;,citation_author=William Jones;,citation_author=Shilpa Garg;,citation_author=Charles Markello;,citation_author=Michael F Lin;,citation_author=Benedict Paten;,citation_author=Richard Durbin;,citation_publication_date=2018-10;,citation_cover_date=2018-10;,citation_year=2018;,citation_fulltext_html_url=http://dx.doi.org/10.1038/nbt.4227;,citation_issue=9;,citation_doi=10.1038/nbt.4227;,citation_volume=36;,citation_language=en;,citation_journal_title=Nature Biotechnology;">
<meta name="citation_reference" content="citation_title=Twelve years of SAMtools and BCFtools;,citation_author=Petr Danecek;,citation_author=James K Bonfield;,citation_author=Jennifer Liddle;,citation_author=John Marshall;,citation_author=Valeriu Ohan;,citation_author=Martin O Pollard;,citation_author=Andrew Whitwham;,citation_author=Thomas Keane;,citation_author=Shane A McCarthy;,citation_author=Robert M Davies;,citation_author=Heng Li;,citation_publication_date=2021-01-29;,citation_cover_date=2021-01-29;,citation_year=2021;,citation_fulltext_html_url=http://dx.doi.org/10.1093/gigascience/giab008;,citation_issue=2;,citation_doi=10.1093/gigascience/giab008;,citation_volume=10;,citation_language=en;,citation_journal_title=GigaScience;">
<meta name="citation_reference" content="citation_title=GraphAligner: rapid and versatile sequence-to-graph alignment;,citation_author=Mikko Rautiainen;,citation_author=Tobias Marschall;,citation_publication_date=2020-09-24;,citation_cover_date=2020-09-24;,citation_year=2020;,citation_fulltext_html_url=http://dx.doi.org/10.1186/s13059-020-02157-2;,citation_issue=1;,citation_doi=10.1186/s13059-020-02157-2;,citation_volume=21;,citation_language=en;,citation_journal_title=Genome Biology;">
<meta name="citation_reference" content="citation_title=Pangenomics enables genotyping of known structural variants in 5202 diverse genomes;,citation_author=Jouni Sirén;,citation_author=Jean Monlong;,citation_author=Xian Chang;,citation_author=Adam M. Novak;,citation_author=Jordan M. Eizenga;,citation_author=Charles Markello;,citation_author=Jonas A. Sibbesen;,citation_author=Glenn Hickey;,citation_author=Pi-Chuan Chang;,citation_author=Andrew Carroll;,citation_author=Namrata Gupta;,citation_author=Stacey Gabriel;,citation_author=Thomas W. Blackwell;,citation_author=Aakrosh Ratan;,citation_author=Kent D. Taylor;,citation_author=Stephen S. Rich;,citation_author=Jerome I. Rotter;,citation_author=David Haussler;,citation_author=Erik Garrison;,citation_author=Benedict Paten;,citation_publication_date=2021-12-17;,citation_cover_date=2021-12-17;,citation_year=2021;,citation_fulltext_html_url=http://dx.doi.org/10.1126/science.abg8871;,citation_issue=6574;,citation_doi=10.1126/science.abg8871;,citation_volume=374;,citation_language=en;,citation_journal_title=Science;">
<meta name="citation_reference" content="citation_title=QUAST: Quality assessment tool for genome assemblies;,citation_author=Alexey Gurevich;,citation_author=Vladislav Saveliev;,citation_author=Nikolay Vyahhi;,citation_author=Glenn Tesler;,citation_publication_date=2013;,citation_cover_date=2013;,citation_year=2013;,citation_issue=8;,citation_volume=29;,citation_journal_title=Bioinformatics;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=BUSCO: Assessing genome assembly and annotation completeness with single-copy orthologs;,citation_author=Felipe A Simão;,citation_author=Robert M Waterhouse;,citation_author=Panagiotis Ioannidis;,citation_author=Evgenia V Kriventseva;,citation_author=Evgeny M Zdobnov;,citation_publication_date=2015;,citation_cover_date=2015;,citation_year=2015;,citation_issue=19;,citation_volume=31;,citation_journal_title=Bioinformatics;,citation_publisher=Oxford University Press;">
<meta name="citation_reference" content="citation_title=Panacus: fast and exact pangenome growth and core size estimation;,citation_author=Luca Parmigiani;,citation_author=Erik Garrison;,citation_author=Jens Stoye;,citation_author=Tobias Marschall;,citation_author=Daniel Doerr;,citation_editor=Janet Kelso;,citation_publication_date=2024-11-29;,citation_cover_date=2024-11-29;,citation_year=2024;,citation_fulltext_html_url=http://dx.doi.org/10.1093/bioinformatics/btae720;,citation_doi=10.1093/bioinformatics/btae720;,citation_language=en;,citation_journal_title=Bioinformatics;">
</head>

<body class="quarto-light">

<header id="title-block-header" class="quarto-title-block default toc-left page-columns page-full">
  <div class="quarto-title-banner page-columns page-full">
    <div class="quarto-title column-body">
      <h1 class="title">Adaptive Introgression Shapes the Pan-genome of Populus Hybrid Zones</h1>
          </div>

    
    <div class="quarto-title-meta-container">
      <div class="quarto-title-meta-column-start">
            <div class="quarto-title-meta-author">
          <div class="quarto-title-meta-heading">Author</div>
          <div class="quarto-title-meta-heading">Affiliation</div>
          
                <div class="quarto-title-meta-contents">
            <p class="author">Baxter Worthing <a href="mailto:baxter.worthing@uvm.edu" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> </p>
          </div>
                <div class="quarto-title-meta-contents">
                    <p class="affiliation">
                        The University of Vermont
                      </p>
                  </div>
                    </div>
        
        <div class="quarto-title-meta">

                      
          
                
              </div>
      </div>
      <div class="quarto-title-meta-column-end quarto-other-formats-target">
      <div class="quarto-alternate-formats"><div class="quarto-title-meta-heading">Other Formats</div><div class="quarto-title-meta-contents"><p><a href="index.docx"><i class="bi bi-file-word"></i>MS Word</a></p></div><div class="quarto-title-meta-contents"><p><a href="index.pdf"><i class="bi bi-file-pdf"></i>PDF (agu)</a></p></div><div class="quarto-title-meta-contents"><p><a href="index-meca.zip" data-meca-link="true"><i class="bi bi-archive"></i>MECA Bundle</a></p></div></div></div>
    </div>

    <div>
      <div class="abstract">
        <div class="block-title">Abstract</div>
        <p>Poplar trees…</p>
      </div>
    </div>


    <div class="quarto-other-links-text-target">
    </div>  </div>
</header><div id="quarto-content" class="page-columns page-rows-contents page-layout-article toc-left">
<div id="quarto-sidebar-toc-left" class="sidebar toc-left">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#introduction" id="toc-introduction" class="nav-link active" data-scroll-target="#introduction"><span class="header-section-number">0.1</span> Introduction</a></li>
  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods"><span class="header-section-number">0.2</span> Methods</a>
  <ul class="collapse">
  <li><a href="#sample-collection-and-whole-genome-sequencing" id="toc-sample-collection-and-whole-genome-sequencing" class="nav-link" data-scroll-target="#sample-collection-and-whole-genome-sequencing"><span class="header-section-number">0.2.1</span> Sample collection and whole genome sequencing</a></li>
  <li><a href="#genome-assembly" id="toc-genome-assembly" class="nav-link" data-scroll-target="#genome-assembly"><span class="header-section-number">0.2.2</span> Genome assembly</a></li>
  <li><a href="#pan-genome-assembly" id="toc-pan-genome-assembly" class="nav-link" data-scroll-target="#pan-genome-assembly"><span class="header-section-number">0.2.3</span> Pan-genome assembly</a></li>
  <li><a href="#genotyping-pan-genomic-variation" id="toc-genotyping-pan-genomic-variation" class="nav-link" data-scroll-target="#genotyping-pan-genomic-variation"><span class="header-section-number">0.2.4</span> Genotyping Pan-genomic Variation</a></li>
  <li><a href="#analysis-of-structural-variation-and-introgression" id="toc-analysis-of-structural-variation-and-introgression" class="nav-link" data-scroll-target="#analysis-of-structural-variation-and-introgression"><span class="header-section-number">0.2.5</span> Analysis of Structural Variation and Introgression</a></li>
  </ul></li>
  <li><a href="#results" id="toc-results" class="nav-link" data-scroll-target="#results"><span class="header-section-number">0.3</span> Results</a>
  <ul class="collapse">
  <li><a href="#whole-genome-sequencing-and-assembly." id="toc-whole-genome-sequencing-and-assembly." class="nav-link" data-scroll-target="#whole-genome-sequencing-and-assembly."><span class="header-section-number">0.3.1</span> Whole genome sequencing and assembly.</a></li>
  <li><a href="#pan-genome-assembly-1" id="toc-pan-genome-assembly-1" class="nav-link" data-scroll-target="#pan-genome-assembly-1"><span class="header-section-number">0.3.2</span> Pan-genome assembly</a></li>
  <li><a href="#pan-genomic-variation" id="toc-pan-genomic-variation" class="nav-link" data-scroll-target="#pan-genomic-variation"><span class="header-section-number">0.3.3</span> Pan-genomic Variation</a></li>
  <li><a href="#sv-and-introgression" id="toc-sv-and-introgression" class="nav-link" data-scroll-target="#sv-and-introgression"><span class="header-section-number">0.3.4</span> SV and Introgression</a></li>
  <li><a href="#tables" id="toc-tables" class="nav-link" data-scroll-target="#tables"><span class="header-section-number">0.3.5</span> Tables</a></li>
  </ul></li>
  <li><a href="#figures" id="toc-figures" class="nav-link" data-scroll-target="#figures"><span class="header-section-number">1</span> Figures</a>
  <ul class="collapse">
  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references">References</a></li>
  </ul></li>
  </ul>
</nav>
</div>
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar zindex-bottom">
</div>
<main class="content quarto-banner-title-block" id="quarto-document-content">



  


<section id="introduction" class="level2" data-number="0.1">
<h2 data-number="0.1" class="anchored" data-anchor-id="introduction"><span class="header-section-number">0.1</span> Introduction</h2>
<p>Hybridization between distinct species can lead to the exchange of genetic variation across species boundaries, a process known as introgression. Introgression is common among inter-fertile species of forest trees, and in many cases, is hypothesized to be an adaptive process, through which genetic variation related to locally-adaptive phenotypes is passed from one species to the other <span class="citation" data-cites="suarez2018adaptive leroy2020adaptive rendon2021adaptive hamilton2016adaptive">(<a href="#ref-suarez2018adaptive" role="doc-biblioref">Suarez-Gonzalez, Lexer, and Cronk 2018</a>; <a href="#ref-leroy2020adaptive" role="doc-biblioref">Leroy et al. 2020</a>; <a href="#ref-rendon2021adaptive" role="doc-biblioref">Rendón-Anaya et al. 2021</a>; <a href="#ref-hamilton2016adaptive" role="doc-biblioref">Hamilton and Miller 2016</a>)</span> Support for this hypothesis requires a clearer understanding of the nature of genetic variation that is exchanged between species, and the degree to which this variation is involved in adaptive processes in natural populations. Characterization of the genetic variation involved in adaptive introgression would shed light on the evolutionary forces that influence the content of admixed genomes and help reveal the loci that underlie both the adaptive traits and genetic incompatibilities that shape species boundaries.</p>
<p>The majority of past research on adaptive introgression has focused on single nucleotide polymorphisms (SNPs) and small indels, but structural variation (SV) is also likely to play a role in adaptive introgression. SV, which includes deletions, duplications, inversions, and translocations of DNA segments, can have a significant impact on gene expression, gene function, and ultimately, adaptive phenotypic variation. For this reason, SV is known to be a major source of genetic variation in many species, and has been implicated in the evolution of adaptive traits in a variety of taxa <span class="citation" data-cites="hamala2021genomic songsomboon2021genomic li2024pan li2023pig">(<a href="#ref-hamala2021genomic" role="doc-biblioref">Hämälä et al. 2021</a>; <a href="#ref-songsomboon2021genomic" role="doc-biblioref">Songsomboon et al. 2021</a>; <a href="#ref-li2024pan" role="doc-biblioref">Y. Li et al. 2024</a>; <a href="#ref-li2023pig" role="doc-biblioref">Z. Li et al. 2023</a>)</span>. As the notion that SV is involved in adaptive evolution gains support, it is only logical to ask is how SV is involved in adaptive introgression. Recent research suggests that SV may play an important role in both adaptive introgression <span class="citation" data-cites="almarri2020population xia2023structural zhang2021pan">(<a href="#ref-almarri2020population" role="doc-biblioref">Almarri et al. 2020</a>; <a href="#ref-xia2023structural" role="doc-biblioref">Xia et al. 2023</a>; <a href="#ref-zhang2021pan" role="doc-biblioref">X. Zhang et al. 2021</a>)</span> and in the maintenance of species boundaries <span class="citation" data-cites="zhang2021important">(<a href="#ref-zhang2021important" role="doc-biblioref">L. Zhang et al. 2021</a>)</span> in natural systems. Introgression between crop varietals and wild relatives has been a key factor in the breeding history of many crops, and recent work shows that SV are often the casual variants involved in this process [<span class="citation" data-cites="gao2019tomato">Gao et al. (<a href="#ref-gao2019tomato" role="doc-biblioref">2019</a>)</span>; <span class="citation" data-cites="kou2020evolutionary">Kou et al. (<a href="#ref-kou2020evolutionary" role="doc-biblioref">2020</a>)</span>; <span class="citation" data-cites="cheng2019frequent">Cheng et al. (<a href="#ref-cheng2019frequent" role="doc-biblioref">2019</a>)</span>; Sun et al., 2020; Qiao et al., 2021; Zanini et al., 2021 ]. While the significant effect of introgressed SV on economically important crop traits is well recognized, far less in known about the relationship between introgression, SV and ecologically important traits in natural hybrid zones.</p>
<p>The study of SV in natural hybrid zones is challenging, as it requires the ability to accurately genotype SV in a large number of admixed individuals. The majority of past research on SV in natural populations has relied on reference genomes that represent only one species, which can lead to reference bias. Reference bias occurs when the genomes of re sequenced individuals contain regions that are highly diverged from the reference genome , causing reads originating from these regions to align incorrectly, or not align at all to the reference genome. This can lead to misinterpretation or under-representation of variation resulting from admixture <span class="citation" data-cites="secomandi2025pangenome">(<a href="#ref-secomandi2025pangenome" role="doc-biblioref">Secomandi et al. 2025</a>)</span>. Reference bias is particularly problematic when studying SV, as large insertions and deletions may be longer than sequencing reads, making variation in their presence and absence (PAV) challenging to detect. Furthermore, SV is often species-specific and, because admixed individuals represent a mosaic of genetic variation from two or more species, the presence of reference bias can obscure the effect that the presence (or absence) of genetic variation has on traits, and can hinder the identification of SV that is involved in introgression.</p>
<p>Recently, an increasing number of studies have relied on pan-genome assembly to overcome reference bias and to accurately genotype SV <span class="citation" data-cites="secomandi2025pangenome gao2019tomato li2023pig li2024pan songsomboon2021genomic">(<a href="#ref-secomandi2025pangenome" role="doc-biblioref">Secomandi et al. 2025</a>; <a href="#ref-gao2019tomato" role="doc-biblioref">Gao et al. 2019</a>; <a href="#ref-li2023pig" role="doc-biblioref">Z. Li et al. 2023</a>; <a href="#ref-li2024pan" role="doc-biblioref">Y. Li et al. 2024</a>; <a href="#ref-songsomboon2021genomic" role="doc-biblioref">Songsomboon et al. 2021</a>)</span>. A pan-genome assembly is a non-redundant collection of sequences originating from multiple individuals. This genetic information can be represented as “nodes” in a pan-genome graph, while the linear sequence of each input genome is stored as a “path” connecting a series of nodes. In a pan-genome graph, each node describes the alignment between at least two sequences, given an expected level of sequence divergence. Pan-genome graphs can capture complex variation that is not present in a single reference genome, and can provide a more accurate representation of the genetic variation present in admixed individuals. Pan-genome assembly has been used to identify and genotype SV related to ecologically and economically important traits in a variety of taxa, and has been shown to be an effective tool for studying the adaptive evolution in natural populations <span class="citation" data-cites="secomandi2025pangenome fang2024fitness">(<a href="#ref-secomandi2025pangenome" role="doc-biblioref">Secomandi et al. 2025</a>; <a href="#ref-fang2024fitness" role="doc-biblioref">Fang and Edwards 2024</a>)</span>. However, to our knowledge, no studies have used pan-genome assembly to examine introgression in the context of forest tree hybrid zones, which are often dynamic and geographically broad. Pan-genome assembly could help overcome the challenges of studying SV in admixed tree genotypes, and could provide new insights into the role of SV in adaptive introgression in natural populations of forest trees.</p>
<p><em>Populus balsamifera</em> and <em>Populus trichocarpa</em> are two species of forest tree that readily interbreed in nature, and early and advanced generation hybrids have been described in hybrid zones located where their ranges overlap (Geraldes et al., 2014; Suarez-Gonzalez et al.,2016; 2018; Chhatre et al.&nbsp;2019). It has previously been hypothesized that hybridization between these two species contributes to an adaptive process, through which genetic variation related to locally-adaptive phenotypes is passed from one species to the other through adaptive introgression (Suarez-Gonzalez et al., 2016; Suarez-Gonzalez et al., 2018). However, studies on inter-specific crosses in <em>Populus</em> have made it clear that introgression is often biased toward particular regions of the genome, particular <em>Populus</em> species, or specific environments (Lexer et al., 2005; Thompson et al., 2010; Meirmans et al., 2017). Therefore, the adaptiveness of introgression between <em>P. balsamifera</em> and <em>P. trichocarpa</em> is dependent on incompatibilities between the genomic variants involved as well as environmental context, two factors which warrant further investigation. The environmental context of adaptive introgression may be of particular importance in the case of <em>P. balsamifera</em> and <em>P. trichocarpa</em>. <em>P. balsamifera</em> is more commonly found in colder, continental climates while <em>P. trichocarpa</em> is more often associated with milder, coastal climates with longer growing seasons (Geraldes et al., 2014; Suarez-Gonzalez et al., 2018). Considering hybrid zones between these species often fall along the boundaries of their ranges, it seems possible that adaptive introgression helps hybrid individuals persist in environments that are outside of the optimum for either parental species. Suarez-Gonzalez et al.&nbsp;(2018) showed support for this hypothesis, finding that introgressed regions in the genomes of hybrids from these zones harbored signs of selection and variation associated with adaptive traits, including phenology. However, the specific adaptive variants within these genomic regions, and the nature of their effects on fitness remain, for the most part, undiscovered.</p>
<p>Here, we take advantage of recent advances in pan-genome assembly methods to produce a pan-genomic reference, comprising 24 diverse haplotypes from <em>P. balsamifera</em>, <em>P. trichocarpa</em> and their hybrids, facilitating unbiased analysis of the sequence variation that is segregating within natural <em>Populus</em> hybrid zones. Using this new pan genomic assembly, we genotype structural variation across 575 individuals sampled from within and outside of natural <em>P. balsamifera</em> and <em>P. trichocarpa</em> hybrid zones. We assess the true extent of genomic diversity present in these species and their hybrids, identifying genomic variation that is not present in the <em>P. trichocarpa</em> reference genome. Furthermore, we describe structural variation involved both in introgression and in genomic divergence between the two species. We shed light on the role that introgression may play in shaping the pan-genome of these species, and the degree to which tree genomes are porous to the inter-specific exchange of structural variant alleles.</p>
</section>
<section id="methods" class="level2" data-number="0.2">
<h2 data-number="0.2" class="anchored" data-anchor-id="methods"><span class="header-section-number">0.2</span> Methods</h2>
<section id="sample-collection-and-whole-genome-sequencing" class="level3" data-number="0.2.1">
<h3 data-number="0.2.1" class="anchored" data-anchor-id="sample-collection-and-whole-genome-sequencing"><span class="header-section-number">0.2.1</span> Sample collection and whole genome sequencing</h3>
<p>In January 2020, we collected dormant branch cuttings from 546 poplar trees along 7 distinct latitudinal transects spanning natural contact zones between Populus trichocarpa and Populus balsamifera <a href="#fig-1" class="quarto-xref">Figure&nbsp;1</a>. Short read whole genome sequencing and subsequent bioinformatic analyses were performed as described in Bolte et al.&nbsp;(2024). Briefly, Genomic DNA libraries for all sampled individuals were constructed at the Duke University Center for Genomic and Computational Biology and sequenced using an Illumina NovaSeq 6000 instrument with an S4 flow cell in 2 × 150 bp format with 64 samples per lane (Illumina Inc., San Diego, USA). De-indexing, QC, trimming adapter sequences, and sequence pre-processing were completed by the sequencing facility. In addition to short read sequencing, we also sequenced a subset of 40 sampled individuals with PacBio HiFi long reads. We harvested tissue from newly-expanded leaves grown under low light conditions to use for high molecular weight DNA extraction. After confirming extraction quality through gel electrophoresis and bioanalysis, we submitted HMW DNA to the University of Maryland Center for Integrative Biosciences Genomics Core Facility for library preparation and sequencing on the the PacBio Sequel system with two samples per SMRT flow cell. We sequenced one <em>P. balsamifera</em> sample (939) for an additional run on a full SMRT cell. We also sequenced this individual with Oxford Nanopore Technology (ONT) platform. We submitted high molecular weight DNA to the Vermont Integrative Genomics Resource Sequencing Center for library preparation and sequencing on XXX flowcell (two runs) and XXX flowcell (one run)</p>
</section>
<section id="genome-assembly" class="level3" data-number="0.2.2">
<h3 data-number="0.2.2" class="anchored" data-anchor-id="genome-assembly"><span class="header-section-number">0.2.2</span> Genome assembly</h3>
<p>Of the 40 individuals sequenced with HiFi long reads, we selected 16 for whole genome assembly <a href="#fig-1" class="quarto-xref">Figure&nbsp;1</a>. These samples ranged from 20 to 35x long read coverage. We performed de-novo genome assembly of the HiFi reads for these samples with HiCanu <span class="citation" data-cites="nurk2020hicanu">(<a href="#ref-nurk2020hicanu" role="doc-biblioref">Nurk et al. 2020</a>)</span>. We set HiCanu parameters as follows: gSize=“400m”, lc=“5”, lcer=“0.055”, ovrlp=“350”, mincov=“9”. To detect potential contaminants in the raw assemblies produced by HiCanu, we used the program Kraken2 to compare the k-mer content of assembled contigs to the “PlusPFP” database of known taxon-specific k-mers representing archaea, bacteria, viral, human, protozoa, fungi and plant taxa <span class="citation" data-cites="lu2022">(<a href="#ref-lu2022" role="doc-biblioref">Lu et al. 2022</a>)</span>. We then used a custom bioawk script to remove any assembled contig that Kraken2 assigned to a taxonomic unit other than <em>Populus</em>, leaving only unassigned or <em>Populus</em>- assigned contigs in each assembly. We assessed the accuracy and completeness of the decontaminated assemblies using QUAST and BUSCO <span class="citation" data-cites="gurevich2013quast simao2015busco">(<a href="#ref-gurevich2013quast" role="doc-biblioref">Gurevich et al. 2013</a>; <a href="#ref-simao2015busco" role="doc-biblioref">Simão et al. 2015</a>)</span>. To further assess the contiguity and quality of these assemblies, we used minimap2 and BWA to map the original HiFi reads and additional Illumina short reads back to each assembled haplotype <span class="citation" data-cites="li2018minimap2 li2010fast">(<a href="#ref-li2018minimap2" role="doc-biblioref">H. Li 2018</a>; <a href="#ref-li2010fast" role="doc-biblioref">H. Li and Durbin 2010</a>)</span>. We passed these alignments to the program CRAQ <span class="citation" data-cites="li2023identification">(<a href="#ref-li2023identification" role="doc-biblioref">K. Li et al. 2023</a>)</span>, which leverages read depth along assembled contigs for quality assessment. We repeated these quality assessment checks at each subsequent stage of genome assembly. Phasing, or the separation and concatenation of contigs belonging to the same parental haplotype is a key step in genome assembly for hybrid individuals, as divergent haplotypes likely contain important genetic information. We used mininmap2 to map reads back to assembled contigs and to map assembled contigs to themselves. We then used the purge haplotigs pipeline (Roach et al., 2018) to split diploid assemblies into two assembled haploytpes for each individual. We used RagTag (Alonge et al.&nbsp;2022) to connect decontaminated, phased contigs into pseudo-chromosomal scaffolds, guided by alignments of assembled contigs to the Nisqually1 <em>P. trichocarpa</em> reference genome. We visually inspected minimap2 alignments of scaffolded assemblies to the reference genome to identify potential scaffolding errors. We used RepeatMasker to annotate and mask repetitive regions and annotated possible coding domains and predicted protein sequences for each assembly with AUGUSTUS (Smit et al., 2015; Hoff et al., 2019).</p>
</section>
<section id="pan-genome-assembly" class="level3" data-number="0.2.3">
<h3 data-number="0.2.3" class="anchored" data-anchor-id="pan-genome-assembly"><span class="header-section-number">0.2.3</span> Pan-genome assembly</h3>
<p>In a pan-genome graph, a non-redundant collection of all input sequences is represented as “nodes”, while the linear sequence of each input genome is stored as a “path” connecting a series of nodes. In this approach to graph construction, each node of the graph w describes the alignment between at least two sequences, given an expected level of sequence divergence. We constructed a pan-genome graph from an all-by all alignment of the 24 assembled haplotypes. Any contigs shorter than 100kb were dropped before alignment. We used wfmash (Guarracino et al., 2021) to perform all-by-all alignments between assembled chromosomes, and seqwish (Garrison and Guarracino, 2022) to project alignments into a graph pan-genome assembly. Pan-genome graphs constructed this way can have highly complex topography, which may hinder downstream applications such a sequence alignment. To combat this issue, we used the program smoothXG to smooth complex variation in these graphs. We also assembled a separate graph specifically for subsequent alignment-based analyses using the minigraph-cactus pipeline <span class="citation" data-cites="hickey2023">(<a href="#ref-hickey2023" role="doc-biblioref">Hickey et al. 2023</a>)</span>. We used the program panacus to asses pan-genome growth and coverage statistics <span class="citation" data-cites="parmigiani2024">(<a href="#ref-parmigiani2024" role="doc-biblioref">Parmigiani et al. 2024</a>)</span>. We used the program ODGI (Guarracino et al., 2022) to perform basic graph quality control and to partition the core ( sequence present in all individuals), shell (sequence variably present or absent across individuals) and singleton sequence (present in only one individuals) content for each graph and individual represented in the graphs. We used the program vg <span class="citation" data-cites="garrison2018">(<a href="#ref-garrison2018" role="doc-biblioref">Garrison et al. 2018</a>)</span> to deconstruct each graph assembly into a VCF containing the variation encapsulated in each graph. We passed this deconstructed VCF file to vcfbub (github.com/pangenome/vcfbub) to collapse nested SV sites into the top-level variant for all SV less than 10kb in length. Using bcftools <span class="citation" data-cites="danecek2021">(<a href="#ref-danecek2021" role="doc-biblioref">Danecek et al. 2021</a>)</span> we split biallelic sites into multiallelic records for this deconstructed vcf.</p>
</section>
<section id="genotyping-pan-genomic-variation" class="level3" data-number="0.2.4">
<h3 data-number="0.2.4" class="anchored" data-anchor-id="genotyping-pan-genomic-variation"><span class="header-section-number">0.2.4</span> Genotyping Pan-genomic Variation</h3>
<p>We aligned sequencing reads from additional samples to pan-genome graph assembly to genotype SNPs and SVs. We aligned PacBio HiFi reads for 40 total individuals (including the 16 used to construct the pan-genome graph) to the minigraph-cactus graph using the program GraphAligner <span class="citation" data-cites="rautiainen2020">(<a href="#ref-rautiainen2020" role="doc-biblioref">Rautiainen and Marschall 2020</a>)</span>. To characterize sequence variation across a broader set of individuals, we also mapped Illumina short reads for 575 individuals to the pan-genome graph using the program giraffe <span class="citation" data-cites="sirén2021">(<a href="#ref-sirén2021" role="doc-biblioref">Sirén et al. 2021</a>)</span>. We then called SV and SNPs from these alignments using the vg call algorithm <span class="citation" data-cites="garrison2018">(<a href="#ref-garrison2018" role="doc-biblioref">Garrison et al. 2018</a>)</span>. We used bcftools to split multialleleic calls into separate records, normalize alleles against the <em>P. trichocarpa</em> reference and merge calls for all samples into one VCF representing SV and SNP genotypes in the <em>P. trichocarpa</em> reference coordinate space.</p>
</section>
<section id="analysis-of-structural-variation-and-introgression" class="level3" data-number="0.2.5">
<h3 data-number="0.2.5" class="anchored" data-anchor-id="analysis-of-structural-variation-and-introgression"><span class="header-section-number">0.2.5</span> Analysis of Structural Variation and Introgression</h3>
<p>We used ADMIXTURE results from Bolte et al (2024) to assign each individual an admixture proportion at K=2 and K=4. The genetic data used for this ADMIXTURE analysis came from SNP calls from mappings of the same illlumina reads used in this study, however they were mapped to the <em>P. trichocarpa</em> reference genome. We used this same genetic data set to perform local ancestry inference (LAI) using LOTR <span class="citation" data-cites="dias2018loter">(<a href="#ref-dias2018loter" role="doc-biblioref">Dias-Alves, Mairal, and Blum 2018</a>)</span>. These ADMIXTURE and LAI data sets were used to visualize and analyze patterns of local and global genomic ancestry throughout this study.</p>
<p>We used ODGI to flatten the Nisqually1 <em>P. trichocarpa</em> reference genome path in the cactus graph. We then used a bed file representation of this path and reference gene annotations to genotype PAV of reference sequence across all paths of the cactus graph. We used R version 4.1.0 to identify core and shell gene sequences from this data and to perform PCA on the PAV genotypes <span class="citation" data-cites="R">(<a href="#ref-R" role="doc-biblioref">R Core Team 2024</a>)</span>. We also ran a similar PAV PCA with the merged vcf representing all 575 sequenced samples in this study, using plink2 <span class="citation" data-cites="chang2015second">(<a href="#ref-chang2015second" role="doc-biblioref">Chang et al. 2015</a>)</span>. We used the -–allele-weights flag to extract the loading of each allele for each SV on the first 3 principal components.</p>
<p>To find SV potentially involved in introgression, we employed custom scripts to identify ancestry blocks along the chromosomes of all 335 admixed individuals in this study. An individual was considered admixed if its k=2 ADMIXTURE score indicated at least 5 percent admixture genome-wide. We used Bedtools <span class="citation" data-cites="quinlan2010bedtools">(<a href="#ref-quinlan2010bedtools" role="doc-biblioref">Quinlan and Hall 2010</a>)</span> to find SV that overlapped ancestry blocks in each admixed individual. We then scored each SV allele based on its prevalence in ancestry blocks that contrasted the average global ancestry of an individual.</p>
<p>For all SV that were outliers in principal component or LAI analyses, we used Bedtools to check for overlap with annotated genes. We then used plantgenie to analyze PFAM enrichment of gene sets <span class="citation" data-cites="sundell2015plant">(<a href="#ref-sundell2015plant" role="doc-biblioref">Sundell et al. 2015</a>)</span>.</p>
</section>
</section>
<section id="results" class="level2" data-number="0.3">
<h2 data-number="0.3" class="anchored" data-anchor-id="results"><span class="header-section-number">0.3</span> Results</h2>
<section id="whole-genome-sequencing-and-assembly." class="level3" data-number="0.3.1">
<h3 data-number="0.3.1" class="anchored" data-anchor-id="whole-genome-sequencing-and-assembly."><span class="header-section-number">0.3.1</span> Whole genome sequencing and assembly.</h3>
<p>Illumina sequencing yielded on average of 8358 MB of sequence, corresponding to an average sequencing depth of 21x. PacBio HiFi sequencing yielded on average 6368 MB of sequence, corresponding to an average sequencing depth of 16x. Sample 939 was sequenced to a depth of roughly 60X HiFi and roughly 20X ONT reads. Table 1 shows summary statistics for the 16 whole genome assemblies included in the pan-genome. Assembly quality and contiguity were generally high, however alternate haplotypes were not always fully assembled (Table 1).</p>
</section>
<section id="pan-genome-assembly-1" class="level3" data-number="0.3.2">
<h3 data-number="0.3.2" class="anchored" data-anchor-id="pan-genome-assembly-1"><span class="header-section-number">0.3.2</span> Pan-genome assembly</h3>
<p>The minigraph-cactus pangenome graph contained a total of 102,907,131 nodes comprising 1.2 Gbp of sequence. Of this, 243 Mbp was present across all individuals (core genome), while 423Mbp was present in some but not all individuals (shell genome). The remaining 537 Mbp represented singleton sequence. <a href="#fig-2" class="quarto-xref">Figure&nbsp;2</a> shows how the size of the sequence classes changes as individuals are added to the pan-genome. <a href="#fig-3" class="quarto-xref">Figure&nbsp;3</a> shows the proportions of core, shell and singleton sequence within each sample. <a href="#fig-4" class="quarto-xref">Figure&nbsp;4</a> shows each genome in the pan-genome graph plotted on the first two principal components of a PCA on PAV of reference sequence.</p>
</section>
<section id="pan-genomic-variation" class="level3" data-number="0.3.3">
<h3 data-number="0.3.3" class="anchored" data-anchor-id="pan-genomic-variation"><span class="header-section-number">0.3.3</span> Pan-genomic Variation</h3>
<p>3708 annotated genes were variably present or absent across the 17 genomes used to construct the pan- =genome. These genes were significantly enriched for various protein families including NB-ARC domain, D-mannose binding lectin, sulfotransferases, cupins, and Cytochrome P450. Of these shell genes, 167 were present in all of the pure <em>P. trichocarpa</em> individuals and none of the pure <em>P. balsamifera</em> samples making up the pan-genome graph. These <em>P. trichocarpa</em> - specific genes were enriched for protein families, including sulfotransferases, serine carboxypeptidases, Mlo family proteins, D-mannose binding lectin, Glycosyl hydrolases and absistic acid (ABA) induced proteins.</p>
<p>The cactus pan-genome graph contained 1,773,427 SV greater than 20bp in length. <a href="#fig-5" class="quarto-xref">Figure&nbsp;5</a> shows the distribution of these variants across the genome. The distribution of frequencies of the non-reference allele for these SV is shown in <a href="#fig-6" class="quarto-xref">Figure&nbsp;6</a>.</p>
<p>The first two principal components of a PCA on PAV genotyped from short read alignments to the pan-genome graph are shown in <a href="#fig-7" class="quarto-xref">Figure&nbsp;7</a>. The loading of each allele for each SV on the first principal component of a PCA on PAV genotyped from short read alignments to the pan-genome graph is shown in <a href="#fig-8" class="quarto-xref">Figure&nbsp;8</a>. The SV that were outliers on PC1 overlapped 1362 annotated genes. These genes were significantly enriched for protein families involved in development and growth (K-Box proteins, KNOX proteins), cold tolerance (DEAD/DEAH box helicases), and phenology (SRF transcription factors).</p>
</section>
<section id="sv-and-introgression" class="level3" data-number="0.3.4">
<h3 data-number="0.3.4" class="anchored" data-anchor-id="sv-and-introgression"><span class="header-section-number">0.3.4</span> SV and Introgression</h3>
<p>We identified 126,973 SV that overlapped ancestry blocks in admixed individuals, and found that 30,721 of these were associated with blocks of either <em>P. trichocarpa</em> or <em>P. balsamifera</em> ancestry in admixed individuals.</p>
</section>
<section id="tables" class="level3" data-number="0.3.5">
<h3 data-number="0.3.5" class="anchored" data-anchor-id="tables"><span class="header-section-number">0.3.5</span> Tables</h3>
<p><a href="images/table.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-1"><img src="images/table.jpg" class="img-fluid"></a></p>
</section>
</section>
<section id="figures" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Figures</h1>
<div id="fig-1" class="quarto-layout-panel">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<div class="quarto-layout-row">
<div class="quarto-layout-cell" style="flex-basis: 50.0%;justify-content: flex-start;">
<a href="images/msamp.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-2" title="Figure&nbsp;1: (Left): sampling locations for 575 individuals used in this study, color indicates ancestry based in ADMIXTURE analysis (Right): A subset of 16 individuals used for whole genome assembly"><img src="images/msamp.jpg" class="img-fluid figure-img"></a>
</div>
</div>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-1-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;1: (Left): sampling locations for 575 individuals used in this study, color indicates ancestry based in ADMIXTURE analysis (Right): A subset of 16 individuals used for whole genome assembly
</figcaption>
</figure>
</div>
<div id="fig-2" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/pg_growth.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-3" title="Figure&nbsp;2: Pan-genome growth curve visualizes the change in core and shell genome size as samples are added"><img src="images/pg_growth.jpg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-2-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;2: Pan-genome growth curve visualizes the change in core and shell genome size as samples are added
</figcaption>
</figure>
</div>
<div id="fig-3" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/pg_content.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-4" title="Figure&nbsp;3: The relative length of the core, shell and singleton portions of the pan-genome for each sample represented"><img src="images/pg_content.jpg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-3-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;3: The relative length of the core, shell and singleton portions of the pan-genome for each sample represented
</figcaption>
</figure>
</div>
<div id="fig-4" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/pav_pca.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-5" title="Figure&nbsp;4: The first two principal components of a PCA on presence/absence variation in the pan-genome. Color indicates ancestry based in ADMIXTURE analysis."><img src="images/pav_pca.jpg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-4-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;4: The first two principal components of a PCA on presence/absence variation in the pan-genome. Color indicates ancestry based in ADMIXTURE analysis.
</figcaption>
</figure>
</div>
<div id="fig-5" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/sv_density.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-6" title="Figure&nbsp;5: The distribution of SV larger than 20bp in length across the genome"><img src="images/sv_density.jpg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-5-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;5: The distribution of SV larger than 20bp in length across the genome
</figcaption>
</figure>
</div>
<div id="fig-6" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/sv_freq.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-7" title="Figure&nbsp;6: The frequency distribution of the non-referenceallele for SV of different size classes"><img src="images/sv_freq.jpg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-6-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;6: The frequency distribution of the non-referenceallele for SV of different size classes
</figcaption>
</figure>
</div>
<div id="fig-7" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/sv_pc1_pc2.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-8" title="Figure&nbsp;7: The first two principal components of a PCA of PAV genotyped from short read alignments to the pan-genome graph. Color indicates ancestry based in ADMIXTURE analysis. Shape indicates which of the latitudinal transects the individual was sampled from."><img src="images/sv_pc1_pc2.jpg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-7-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;7: The first two principal components of a PCA of PAV genotyped from short read alignments to the pan-genome graph. Color indicates ancestry based in ADMIXTURE analysis. Shape indicates which of the latitudinal transects the individual was sampled from.
</figcaption>
</figure>
</div>
<div id="fig-8" class="quarto-float quarto-figure quarto-figure-center anchored">
<figure class="quarto-float quarto-float-fig figure">
<div aria-describedby="fig-8-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
<a href="images/pc1_loading.jpg" class="lightbox" data-gallery="quarto-lightbox-gallery-9" title="Figure&nbsp;8: The loading of each allele for each SV on the first principle component of a PCA on PAV genotyped from short read alignments to the pan-genome graph. Red points are indicate P. trichocarpa reference alleles, blue points indicate non-reference alleles."><img src="images/pc1_loading.jpg" class="img-fluid figure-img"></a>
</div>
<figcaption class="quarto-float-caption-bottom quarto-float-caption quarto-float-fig" id="fig-8-caption-0ceaefa1-69ba-4598-a22c-09a6ac19f8ca">
Figure&nbsp;8: The loading of each allele for each SV on the first principle component of a PCA on PAV genotyped from short read alignments to the pan-genome graph. Red points are indicate P. trichocarpa reference alleles, blue points indicate non-reference alleles.
</figcaption>
</figure>
</div>
<section id="references" class="level2 unnumbered">
<h2 class="unnumbered anchored" data-anchor-id="references">References</h2>
<div id="refs" class="references csl-bib-body hanging-indent" data-entry-spacing="0" role="list">
<div id="ref-almarri2020population" class="csl-entry" role="listitem">
Almarri, Mohamed A, Anders Bergström, Javier Prado-Martinez, Fengtang Yang, Beiyuan Fu, Alistair S Dunham, Yuan Chen, Matthew E Hurles, Chris Tyler-Smith, and Yali Xue. 2020. <span>“Population Structure, Stratification, and Introgression of Human Structural Variation.”</span> <em>Cell</em> 182 (1): 189–99.
</div>
<div id="ref-chang2015second" class="csl-entry" role="listitem">
Chang, Christopher C, Carson C Chow, Laurent CAM Tellier, Shashaank Vattikuti, Shaun M Purcell, and James J Lee. 2015. <span>“Second-Generation PLINK: Rising to the Challenge of Larger and Richer Datasets.”</span> <em>Gigascience</em> 4 (1): s13742–015.
</div>
<div id="ref-cheng2019frequent" class="csl-entry" role="listitem">
Cheng, Hong, Jing Liu, Jia Wen, Xiaojun Nie, Luohao Xu, Ningbo Chen, Zhongxing Li, et al. 2019. <span>“Frequent Intra-and Inter-Species Introgression Shapes the Landscape of Genetic Variation in Bread Wheat.”</span> <em>Genome Biology</em> 20: 1–16.
</div>
<div id="ref-danecek2021" class="csl-entry" role="listitem">
Danecek, Petr, James K Bonfield, Jennifer Liddle, John Marshall, Valeriu Ohan, Martin O Pollard, Andrew Whitwham, et al. 2021. <span>“Twelve Years of SAMtools and BCFtools.”</span> <em>GigaScience</em> 10 (2). <a href="https://doi.org/10.1093/gigascience/giab008">https://doi.org/10.1093/gigascience/giab008</a>.
</div>
<div id="ref-dias2018loter" class="csl-entry" role="listitem">
Dias-Alves, Thomas, Julien Mairal, and Michael GB Blum. 2018. <span>“Loter: A Software Package to Infer Local Ancestry for a Wide Range of Species.”</span> <em>Molecular Biology and Evolution</em> 35 (9): 2318–26.
</div>
<div id="ref-fang2024fitness" class="csl-entry" role="listitem">
Fang, Bohao, and Scott V Edwards. 2024. <span>“Fitness Consequences of Structural Variation Inferred from a House Finch Pangenome.”</span> <em>Proceedings of the National Academy of Sciences</em> 121 (47): e2409943121.
</div>
<div id="ref-gao2019tomato" class="csl-entry" role="listitem">
Gao, Lei, Itay Gonda, Honghe Sun, Qiyue Ma, Kan Bao, Denise M Tieman, Elizabeth A Burzynski-Chang, et al. 2019. <span>“The Tomato Pan-Genome Uncovers New Genes and a Rare Allele Regulating Fruit Flavor.”</span> <em>Nature Genetics</em> 51 (6): 1044–51.
</div>
<div id="ref-garrison2018" class="csl-entry" role="listitem">
Garrison, Erik, Jouni Sirén, Adam M Novak, Glenn Hickey, Jordan M Eizenga, Eric T Dawson, William Jones, et al. 2018. <span>“Variation Graph Toolkit Improves Read Mapping by Representing Genetic Variation in the Reference.”</span> <em>Nature Biotechnology</em> 36 (9): 875–79. <a href="https://doi.org/10.1038/nbt.4227">https://doi.org/10.1038/nbt.4227</a>.
</div>
<div id="ref-gurevich2013quast" class="csl-entry" role="listitem">
Gurevich, Alexey, Vladislav Saveliev, Nikolay Vyahhi, and Glenn Tesler. 2013. <span>“QUAST: Quality Assessment Tool for Genome Assemblies.”</span> <em>Bioinformatics</em> 29 (8): 1072–75.
</div>
<div id="ref-hamala2021genomic" class="csl-entry" role="listitem">
Hämälä, Tuomas, Eric K Wafula, Mark J Guiltinan, Paula E Ralph, Claude W Depamphilis, and Peter Tiffin. 2021. <span>“Genomic Structural Variants Constrain and Facilitate Adaptation in Natural Populations of Theobroma Cacao, the Chocolate Tree.”</span> <em>Proceedings of the National Academy of Sciences</em> 118 (35): e2102914118.
</div>
<div id="ref-hamilton2016adaptive" class="csl-entry" role="listitem">
Hamilton, Jill A, and Joshua M Miller. 2016. <span>“Adaptive Introgression as a Resource for Management and Genetic Conservation in a Changing Climate.”</span> <em>Conservation Biology</em> 30 (1): 33–41.
</div>
<div id="ref-hickey2023" class="csl-entry" role="listitem">
Hickey, Glenn, Jean Monlong, Jana Ebler, Adam M. Novak, Jordan M. Eizenga, Yan Gao, Haley J. Abel, et al. 2023. <span>“Pangenome Graph Construction from Genome Alignments with Minigraph-Cactus.”</span> <em>Nature Biotechnology</em> 42 (4): 663–73. <a href="https://doi.org/10.1038/s41587-023-01793-w">https://doi.org/10.1038/s41587-023-01793-w</a>.
</div>
<div id="ref-kou2020evolutionary" class="csl-entry" role="listitem">
Kou, Yixuan, Yi Liao, Tuomas Toivainen, Yuanda Lv, Xinmin Tian, JJ Emerson, Brandon S Gaut, and Yongfeng Zhou. 2020. <span>“Evolutionary Genomics of Structural Variation in Asian Rice (Oryza Sativa) Domestication.”</span> <em>Molecular Biology and Evolution</em> 37 (12): 3507–24.
</div>
<div id="ref-leroy2020adaptive" class="csl-entry" role="listitem">
Leroy, Thibault, Jean-Marc Louvet, Céline Lalanne, Grégoire Le Provost, Karine Labadie, Jean-Marc Aury, Sylvain Delzon, Christophe Plomion, and Antoine Kremer. 2020. <span>“Adaptive Introgression as a Driver of Local Adaptation to Climate in European White Oaks.”</span> <em>New Phytologist</em> 226 (4): 1171–82.
</div>
<div id="ref-li2018minimap2" class="csl-entry" role="listitem">
Li, Heng. 2018. <span>“Minimap2: Pairwise Alignment for Nucleotide Sequences.”</span> <em>Bioinformatics</em> 34 (18): 3094–3100.
</div>
<div id="ref-li2010fast" class="csl-entry" role="listitem">
Li, Heng, and Richard Durbin. 2010. <span>“Fast and Accurate Long-Read Alignment with Burrows–Wheeler Transform.”</span> <em>Bioinformatics</em> 26 (5): 589–95.
</div>
<div id="ref-li2023identification" class="csl-entry" role="listitem">
Li, Kunpeng, Peng Xu, Jinpeng Wang, Xin Yi, and Yuannian Jiao. 2023. <span>“Identification of Errors in Draft Genome Assemblies at Single-Nucleotide Resolution for Quality Assessment and Improvement.”</span> <em>Nature Communications</em> 14 (1): 6556.
</div>
<div id="ref-li2024pan" class="csl-entry" role="listitem">
Li, Yancan, Jun Yao, Huiling Sang, Quangui Wang, Long Su, Xiaomeng Zhao, Zhenyu Xia, et al. 2024. <span>“Pan-Genome Analysis Highlights the Role of Structural Variation in the Evolution and Environmental Adaptation of Asian Honeybees.”</span> <em>Molecular Ecology Resources</em> 24 (2): e13905.
</div>
<div id="ref-li2023pig" class="csl-entry" role="listitem">
Li, Zhengcao, Xiaohong Liu, Chen Wang, Zhenyang Li, Bo Jiang, Ruifeng Zhang, Lu Tong, et al. 2023. <span>“The Pig Pangenome Provides Insights into the Roles of Coding Structural Variations in Genetic Diversity and Adaptation.”</span> <em>Genome Research</em> 33 (10): 1833–47.
</div>
<div id="ref-lu2022" class="csl-entry" role="listitem">
Lu, Jennifer, Natalia Rincon, Derrick E. Wood, Florian P. Breitwieser, Christopher Pockrandt, Ben Langmead, Steven L. Salzberg, and Martin Steinegger. 2022. <span>“Metagenome Analysis Using the Kraken Software Suite.”</span> <em>Nature Protocols</em> 17 (12): 2815–39. <a href="https://doi.org/10.1038/s41596-022-00738-y">https://doi.org/10.1038/s41596-022-00738-y</a>.
</div>
<div id="ref-nurk2020hicanu" class="csl-entry" role="listitem">
Nurk, Sergey, Brian P Walenz, Arang Rhie, Mitchell R Vollger, Glennis A Logsdon, Robert Grothe, Karen H Miga, Evan E Eichler, Adam M Phillippy, and Sergey Koren. 2020. <span>“HiCanu: Accurate Assembly of Segmental Duplications, Satellites, and Allelic Variants from High-Fidelity Long Reads.”</span> <em>Genome Research</em> 30 (9): 1291–1305.
</div>
<div id="ref-parmigiani2024" class="csl-entry" role="listitem">
Parmigiani, Luca, Erik Garrison, Jens Stoye, Tobias Marschall, and Daniel Doerr. 2024. <span>“Panacus: Fast and Exact Pangenome Growth and Core Size Estimation.”</span> Edited by Janet Kelso. <em>Bioinformatics</em>, November. <a href="https://doi.org/10.1093/bioinformatics/btae720">https://doi.org/10.1093/bioinformatics/btae720</a>.
</div>
<div id="ref-quinlan2010bedtools" class="csl-entry" role="listitem">
Quinlan, Aaron R, and Ira M Hall. 2010. <span>“BEDTools: A Flexible Suite of Utilities for Comparing Genomic Features.”</span> <em>Bioinformatics</em> 26 (6): 841–42.
</div>
<div id="ref-R" class="csl-entry" role="listitem">
R Core Team. 2024. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/">https://www.R-project.org/</a>.
</div>
<div id="ref-rautiainen2020" class="csl-entry" role="listitem">
Rautiainen, Mikko, and Tobias Marschall. 2020. <span>“GraphAligner: Rapid and Versatile Sequence-to-Graph Alignment.”</span> <em>Genome Biology</em> 21 (1). <a href="https://doi.org/10.1186/s13059-020-02157-2">https://doi.org/10.1186/s13059-020-02157-2</a>.
</div>
<div id="ref-rendon2021adaptive" class="csl-entry" role="listitem">
Rendón-Anaya, Martha, Jonathan Wilson, Sæmundur Sveinsson, Aleksey Fedorkov, Joan Cottrell, Mark ES Bailey, Dainis Ruņǵis, et al. 2021. <span>“Adaptive Introgression Facilitates Adaptation to High Latitudes in European Aspen (Populus Tremula l.).”</span> <em>Molecular Biology and Evolution</em> 38 (11): 5034–50.
</div>
<div id="ref-secomandi2025pangenome" class="csl-entry" role="listitem">
Secomandi, Simona, Guido Roberto Gallo, Riccardo Rossi, Carlos Rodrı́guez Fernandes, Erich D Jarvis, Andrea Bonisoli-Alquati, Luca Gianfranceschi, and Giulio Formenti. 2025. <span>“Pangenome Graphs and Their Applications in Biodiversity Genomics.”</span> <em>Nature Genetics</em>, 1–14.
</div>
<div id="ref-simao2015busco" class="csl-entry" role="listitem">
Simão, Felipe A, Robert M Waterhouse, Panagiotis Ioannidis, Evgenia V Kriventseva, and Evgeny M Zdobnov. 2015. <span>“BUSCO: Assessing Genome Assembly and Annotation Completeness with Single-Copy Orthologs.”</span> <em>Bioinformatics</em> 31 (19): 3210–12.
</div>
<div id="ref-sirén2021" class="csl-entry" role="listitem">
Sirén, Jouni, Jean Monlong, Xian Chang, Adam M. Novak, Jordan M. Eizenga, Charles Markello, Jonas A. Sibbesen, et al. 2021. <span>“Pangenomics Enables Genotyping of Known Structural Variants in 5202 Diverse Genomes.”</span> <em>Science</em> 374 (6574). <a href="https://doi.org/10.1126/science.abg8871">https://doi.org/10.1126/science.abg8871</a>.
</div>
<div id="ref-songsomboon2021genomic" class="csl-entry" role="listitem">
Songsomboon, Kittikun, Zachary Brenton, James Heuser, Stephen Kresovich, Nadia Shakoor, Todd Mockler, and Elizabeth A Cooper. 2021. <span>“Genomic Patterns of Structural Variation Among Diverse Genotypes of Sorghum Bicolor and a Potential Role for Deletions in Local Adaptation.”</span> <em>G3</em> 11 (7): jkab154.
</div>
<div id="ref-suarez2018adaptive" class="csl-entry" role="listitem">
Suarez-Gonzalez, Adriana, Christian Lexer, and Quentin CB Cronk. 2018. <span>“Adaptive Introgression: A Plant Perspective.”</span> <em>Biology Letters</em> 14 (3): 20170688.
</div>
<div id="ref-sundell2015plant" class="csl-entry" role="listitem">
Sundell, David, Chanaka Mannapperuma, Sergiu Netotea, Nicolas Delhomme, Yao-Cheng Lin, Andreas Sjodin, Yves Van de Peer, Stefan Jansson, Torgeir R Hvidsten, and Nathaniel R Street. 2015. <span>“The Plant Genome Integrative Explorer Resource: PlantGenIE. Org.”</span>
</div>
<div id="ref-xia2023structural" class="csl-entry" role="listitem">
Xia, Xiaoting, Fengwei Zhang, Shuang Li, Xiaoyu Luo, Lixin Peng, Zheng Dong, Hubert Pausch, et al. 2023. <span>“Structural Variation and Introgression from Wild Populations in East Asian Cattle Genomes Confer Adaptation to Local Environment.”</span> <em>Genome Biology</em> 24 (1): 211.
</div>
<div id="ref-zhang2021important" class="csl-entry" role="listitem">
Zhang, Linyi, Radka Reifová, Zuzana Halenková, and Zachariah Gompert. 2021. <span>“How Important Are Structural Variants for Speciation?”</span> <em>Genes</em> 12 (7): 1084.
</div>
<div id="ref-zhang2021pan" class="csl-entry" role="listitem">
Zhang, Xiaohui, Tongjin Liu, Jinglei Wang, Peng Wang, Yang Qiu, Wei Zhao, Shuai Pang, et al. 2021. <span>“Pan-Genome of Raphanus Highlights Genetic Variation and Introgression Among Domesticated, Wild, and Weedy Radishes.”</span> <em>Molecular Plant</em> 14 (12): 2032–55.
</div>
</div>
</section>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->
<script>var lightboxQuarto = GLightbox({"closeEffect":"zoom","descPosition":"bottom","loop":false,"openEffect":"zoom","selector":".lightbox"});
(function() {
  let previousOnload = window.onload;
  window.onload = () => {
    if (previousOnload) {
      previousOnload();
    }
    lightboxQuarto.on('slide_before_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      const href = trigger.getAttribute('href');
      if (href !== null) {
        const imgEl = window.document.querySelector(`a[href="${href}"] img`);
        if (imgEl !== null) {
          const srcAttr = imgEl.getAttribute("src");
          if (srcAttr && srcAttr.startsWith("data:")) {
            slideConfig.href = srcAttr;
          }
        }
      } 
    });
  
    lightboxQuarto.on('slide_after_load', (data) => {
      const { slideIndex, slideNode, slideConfig, player, trigger } = data;
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(slideNode);
      }
    });
  
  };
  
})();
          </script>




</body></html>